cmd/mist-api-connector/mist-api-connector.go

```diff
@@ -40,10 +39,9 @@ func main() {
 	sendAudio := fs.String("send-audio", "record", "when should we send audio?  {always|never|record}")
 	apiToken := fs.String("api-token", "", "Token of the Livepeer API to be used by the Mist server")
 	apiServer := fs.String("api-server", livepeer.ACServer, "Livepeer API server to use")
-	consulURI := fs.String("consul", "", "Base URL to access Consul (for example: http://localhost:8500)")
-	consulPrefix := fs.String("consul-prefix", "", "Prefix to be prepended to all created consul routes e.g. 'nyc-'")
-	playbackDomain := fs.String("playback-domain", "", "regex of domain to create consul routes for (ex: playback.livepeer.live)")
-	mistURL := fs.String("consul-mist-url", "", "external URL of this Mist instance (to be put in Consul) (ex: https://mist-server-0.livepeer.live)")
+	routePrefix := fs.String("route-prefix", "", "Prefix to be prepended to all created routes e.g. 'nyc-'")
+	playbackDomain := fs.String("playback-domain", "", "regex of domain to create routes for (ex: playback.livepeer.live)")
```

Comment:
Since this is a regex, should it be like `playback\.livepeer\.live`? Guess it doesn't matter much in the example, but I wonder if we set it correctly in prod

Reply:
Trafeik's regex thingy is really weird and not well documentedd, I think only things that are within curly braces get parsed as regexes? ["HostRegexp and Path accept an expression with zero or more groups enclosed by curly braces. Named groups can be like {name:pattern} that matches the given regexp pattern or like {name} that matches anything until the next dot."](https://doc.traefik.io/traefik/routing/routers/) So our current prod value is `{prefix:.*}{domain:(playback-redacted|cdn)}.livepeer.com` and I believe that works as intended